src/model/paths.mli

```diff
@@ -16,6 +16,8 @@
 
```

Comment:
All these diffs make the `Paths` module even more annoying. What do you think of getting rid of it ?
It is not abstracting anything from `Paths_types`, it's just adding wrapper modules, which are not used much and could be written inline every time we want a `Map` or a `Set`.